src/libs/highlevel/README.md

```diff
@@ -0,0 +1,154 @@
+
+# High-Level API
+
+## Introduction
+
+The goal of the High-Level API is to increase the usability of libelektra for developers who want to integrate Elektra into their applications. Projects usually do not want to use low-level APIs. `KDB` and `KeySet` are useful for plugins and to implement APIs but not to be directly used in applications. The High-Level API should be extremely easy to get started with and at the same time it should be hard to use it in a wrong way. This tutorial gives an introduction for developers who want to elektrify their application using the High-Level API.
+
+## Setup
+
+First you have to add `elektra-highlevel` to the linked libraries of your application. To be able to use it in your source file, just include the main header with `#include "elektra.h"` at the top of your file.
+
+## Core Concepts
+
+### Struct `Elektra`
+`Elektra` is the handle you use to access the underlying KDB. All key-value read and write operations expect this handle to be passed as in as a parameter. To create the handle, you can simply write:
+
+```c
+ElektraError * error = NULL;
+Elektra * elektra = elektraOpen ("/sw/yourApplication", &error);
```

Comment:
I think the user gets the point. Otherwise he would ask himself what the NULL parameter could be.